src/main/webapp/index.html

```diff
@@ -27,7 +27,8 @@
       <button class="dropbtn">Menu</button>
       <div class="dropdown-content">
       <ul id="navigation">
-        <li><a href="/">Home</a></li>
+        <li><a href="/feed.html">Feed</a></li>
+        <li><a href="/stats.html">Stats</a></li>
```

Comment:
same as above - not sure if the link to the user page is automatically added?

Reply:
On this page it shows up automatically